test/cluster.go

```diff
@@ -92,3 +105,58 @@ func AssertEqualIdentityClusters(t *testing.T, expected, actual *repository.Iden
 	assert.Equal(t, expected.IdentityID, actual.IdentityID)
 	assert.Equal(t, expected.ClusterID, actual.ClusterID)
 }
+
+type OSOClusterConfig struct {
+	Clusters []configuration.OSOCluster
+}
+
+func GetClustersFromConfigFile(filePath string) ([]repository.Cluster, error) {
```

Comment:
Why do you need to load clusters from a config file? Why not to use config.GetOSOClusters()?

Reply:
This is used in tests where we need  loaded `Cluster`  from config to compare it with saved in DB so that we don't have hard coded clusters in tests

Reply:
Sure, but config.GetOSOClusters() will return the same clusters (it will load it from the config file), won't it?
Why do you need to duplicate the config file loading in the test?

Reply:
I missed that, my bad. Addressed in eb19a6117225988cd2e90aaf736fa7e15b910742